plugins/catalog-backend/src/service/createRouter.test.ts

```diff
@@ -540,6 +540,13 @@ describe('createRouter readonly disabled', () => {
         entityRefs: [entityRef],
         fields: expect.any(Function),
         credentials: mockCredentials.user(),
+        filter: {
+          anyOf: [
```

Comment:
ooof this unnecessary anyOf+oneOf nesting of single-element things stands out like a sore thumb :D well, some other day

Reply:
Yeah, it's nice 😆